releasenotes/notes/linear-binary-matrix-utils-rust-c48b5577749c34ab.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features_synthesis:
+  - |
+    Port internal binary matrix utils from Python to rust, including
```

Comment:
Is it `rust` or `Rust`? 🙂 Also depending on how we remain for the input types of the function, we should mention if only boolean matrices are supported now.